### DISCUSSION TITLE
Does anyone know what the nutcracker means?

### DISCUSSION THREAD

**Discussion by ZcBaxter** [DaXcess/LCVR#210] [Q&A] [answered]
I found a nutcracker when loading up the game in VR mode. It was to my left and pointing its gun at me. The only mods I had was the VR mod, BepinEx and the Yippee bug mod. Does anyone know what this could mean?

**Comment by DaXcess** [comment, marked as answer]
The Nutcracker triangulates the position to your house and is out to get you next Christmas
